client/simple/src/less/weather.less

```diff
@@ -0,0 +1,34 @@
+#answers .weather {
+  summary {
+    display: block;
+  }
+
+  div.summary {
+    margin: 0;
+    padding: 0.5rem 1rem;
+    background-color: var(--color-header-background);
```

Comment:
Maybe we should add some rounded borders to the summary to make it fit in better?

```suggestion
    background-color: var(--color-header-background);
    .rounded-corners-tiny;
```
![2025-05-25T11:18:27,893244402+02:00](https://github.com/user-attachments/assets/6f201588-4823-4487-a4d0-8304ca3d83a3)

vs

![2025-05-25T11:26:27,171415336+02:00](https://github.com/user-attachments/assets/92c29225-7fdc-4a0c-b9f4-b9376c3c73a8)